policyengine_us/parameters/gov/states/de/tax/income/subtractions/exclusions/pension/income_sources.yaml

```diff
@@ -0,0 +1,19 @@
+description: Delaware counts these income sources for the pension exclusion.
+values:
+  2021-01-01:
+    - dividend_income
+    - capital_gains
+    - taxable_interest_income
+    - rental_income
+# qualified retirement plans (IRC Sec. 4974), such as IRA, 401(k), Keogh plans, 
```

Comment:
I think we have variables for these, like `pension_income` (or maybe `taxable_pension_income` depending on the law)

Reply:
why resolved?